mycore-base/src/main/java/org/mycore/common/content/transformer/MCRXsonifyTransformer.java

```diff
@@ -1,9 +1,29 @@
 package org.mycore.common.content.transformer;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.node.ObjectNode;
-import jakarta.inject.Singleton;
+import static org.mycore.xsonify.serialize.SerializerSettings.AdditionalNamespaceDeclarationStrategy;
```

Comment:
Could you undo the import statement reorderung?

Reply:
"My eclipse" (configured like documented on MyCoRe website)
 sorts static imports before "normal" imports, so the change is O.K. for me.

Reply:
Same here, I also use the eclipse code formatter for intellij. And now it seems correct.

Another unrelated question @toKrause 
I still have this disabled code in the class

```
    /**
     * Map of namespaces. This setting is required for the json2xml serialisation process if the
     * {@link SerializerSettings#namespaceDeclaration()} is set to <b>OMIT</b>.
     * TODO: Map property doesn't seem to work right now
     */
    //@MCRProperty(name = "Namespaces", required = false)
    //public Map<String, String> namespaces;

```

Should this work now?

Reply:
A feature like that has not yet been added. I've just made [MCR-3362](https://github.com/MyCoRe-Org/mycore/pull/2451) that would allow for this. Code here would be

``` java
@MCRProperty(name = "Namespaces.*", required = false)
public Map<String, String> namespaces;
```

and contain keys with the `Namespaces.` prefix already removed.

Reply:
Ok wow cool, das ging schnell. Dank dir :)